Testreglar/1.4.5/App/app-1.4.5a.json

```diff
@@ -0,0 +1,162 @@
+			"type": "tekst",
+			"label": "Bilde av tekst:",
+			"multilinje": true,
+			"ruting": {
+				"alle": {
+					"type": "gaaTil",
+					"steg": "3.2"
+				}
+			}
+		},
+		{
+			"stegnr": "3.2",
+			"spm": "Er bruken av bilde av tekst avgjerande for informasjonen som blir formidla?",
+			"ht": "<blockquote><em>Avgjørende (essential): Dersom avgjørende elementer fjernes, vil det medføre en grunnleggende endring av informasjonen eller funksjonaliteten i innholdet, <strong>og</strong> informasjonen og funksjonaliteten vil ikke kunne gjenskapes på en annen måte som samsvarer med kravene</em></blockquote><p>Vurder om ein bestemt presentasjon er avgjerande (essensial) for informasjonen som blir formidla. </p><p>Eksempel: </p><ul><li>logo (tekst som utgjør en del av en logo eller et varemerkenavn), jamfør merknad til suksesskriteriet</li><li>bilde av eit produkt</li><li>tekst i grafiske element, for eksempel diagram og grafar </li></ul>",
+			"type": "jaNei",
+			"ruting": {
+				"nei": {
+					"type": "gaaTil",
+					"steg": "3.3"
+				},
+				"ja": {
+					"type": "ikkjeForekomst",
+					"utfall": "Bruk av bilde av tekst er avgjerande, og omfatta av unntaket i kravet."
+				}
+			},
+			"kilde": [
+				"wcag:ordliste"
+			]
+		},
+		{
+			"stegnr": "3.3",
+			"spm": "Er det mogleg å tilpasse bilde av tekst, til brukaren sine krav?",
+			"ht": "<p>Slik testar du:</p><ul><li>Sjekk om følgande i bilde av tekst kan endrast, jamfør definisjonen av Visuelt tilpasset</li></ul><ul><li style=\"list-style-type: none;\"><ul><li>skrifttype</li><li>skriftstørrelse</li><li>farge på tekst</li><li>bakgrunn</li></ul></li></ul><ul><li>Alle punkta over skal oppfylte.</li></ul><p><strong>Merk:</strong></p><ul><li>Funksjonaliteten kan gis via ein mekanisme.</li><li>Her har vi ikkje ein bestemt framgangsmåte for verifikasjon. Difor er svaralternativet \"Ikkje mogleg å verifisere\" tatt med.</li></ul>",
+			"type": "radio",
+			"svarArray": [
+				"Ja",
+				"Nei",
+				"Ikkje mogleg å verifisere"
+			],
+			"ruting": {
+				"alt0": {
+					"type": "avslutt",
+					"fasit": "Ja",
+					"utfall": "Bilde av tekst kan visuelt tilpassast brukaren sine krav."
+				},
+				"alt1": {
+					"type": "gaaTil",
+					"steg": "3.4"
+				},
+				"alt2": {
+					"type": "avslutt",
+					"fasit": "Ikkje testbart",
+					"utfall": "Det er ikkje mogleg å verifisera om bilde av tekst kan visuelt tilpassast brukaren sine krav. "
+				}
+			},
+			"kilde": [
+				"wcag:ordliste"
+			]
+		},
+		{
+			"stegnr": "3.4",
+			"spm": "Går informasjon/funksjonalitet i presentasjonen tapt, viss bilde av tekst blir koda som tekst?",
+			"ht": "<p>Eksempel på situasjonar der bruk av bilde av tekst er nødvendig:</p><ul><li>Det ikkje er mogleg å formatere teksten for å oppnå same effekt.</li><li>Det ikkje er mogleg å presentere effekten i dei mest brukte app-visninger.</li><li>Presentasjonen krev bruk av teknikkar/teknologi som ikkje oppfyller WCAG.</li></ul>",
```

Comment:
Det er ikkje **teknisk** er mogleg å formatere teksten for å oppnå same effekt. -> Dersom det er det som er tenkt vurdert.